src/future.js

```diff
@@ -59,7 +59,13 @@ export function isFuture(x){
   return x instanceof Future || type(x) === $$type;
 }
 
+// Compliance to sanctuary-type-identifiers version 1 and 2.
+// In order to prevent sanctuary-type-identifiers version 3 from identifying
+// Future as a member of $$type, we ensure that Future.constructor.prototype
+// is equal to Future.
```

Comment:
`$$type` is not of type `Type` but of type `String`. :zany_face:

I think `$$type` should be replaced with `Future a b` in the comment above.

The comment could begin with _To prevent_ rather than _In order to prevent_ without any loss of meaning.

Reply:
I'm don't like bringing the type parameters into the picture, because they have nothing to do with sanctuary-type-identifers. I rephrased that part though. What do you think of the current phrasing?